app/routes/fade.js

```diff
@@ -1,205 +1,136 @@
 var querystring = require("querystring"),
-  path = require("path"),
-  error = require(path.join(__dirname, "/../../lib/error.js")),
-  globals = require(path.join(__dirname, "/../../config/globals.js")),
-  mailer = require(path.join(__dirname, "/../../lib/mailer.js")),
-  helpers = require(path.join(__dirname, "/../../lib/helpers.js")),
-  hpcsocket = require(path.join(__dirname, "/../../lib/hpcsocket.js")),
+  error = require(__dirname + " /../../lib/error.js"),
+  globals = require(__dirname + "/../../config/globals.js"),
+  mailer = require(__dirname + "/../../lib/mailer.js"),
+  helpers = require(__dirname + "/../../lib/helpers.js"),
+  hpcsocket = require(__dirname + "/../../lib/hpcsocket.js"),
   fs = require("fs"),
-  winston = require("winston"),
   logger = require("../../lib/logger");
 
 var mongoose = require("mongoose"),
   Msa = mongoose.model("Msa"),
   Sequences = mongoose.model("Sequences"),
   PartitionInfo = mongoose.model("PartitionInfo"),
-  Fade = mongoose.model("Fade");
+  FADE = mongoose.model("Fade");
 
-exports.createForm = function(req, res) {
-  res.render("fade/upload_msa.ejs");
-};
-
-exports.uploadFile = function(req, res) {
-  var PROTEIN_GENCODE = -2;
-  var data = req.body;
-  var fn = req.files.files.file;
+var redis = require("redis"),
+  client = redis.createClient({ host: "localhost", port: 6379 });
```

Comment:
Maybe I'm a bad example in other parts of the code, but the redis port should be configurable instead of hardcoded.

Reply:
As well as the host.

Reply:
Makes sense. My last commit addressed this. Update http://newmethods.datamonkey.org/ to ensure it works.

Reply:
I updated http://newmethods.datamonkey.org/ and successfully ran a SLAC and FADE job to make sure the change worked 👍